package.json

```diff
@@ -76,12 +77,19 @@
     "@storybook/manager-webpack5": "6.3.12",
     "@storybook/preset-scss": "^1.0.3",
     "@storybook/react": "6.3.12",
+    "@swc/core": "^1.2.133",
+    "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "@zeit/next-less": "^1.0.1",
+    "@zeit/next-sass": "^1.0.1",
+    "@zeit/next-stylus": "^1.0.1",
     "babel-loader": "^8.2.2",
     "babel-preset-react-app": "^10.0.0",
     "chromatic": "^6.0.4",
+    "critters": "^0.0.16",
```

Comment:
Where is this being used?

Reply:
Are these all being used?

Reply:
All these are used for fixing the dependencies for bundling Next.js, which is the dependencies they have left out. And yes these should be moved to devDependencies